impl_core/src/main/java/io/opencensus/implcore/trace/export/ExportComponentImpl.java

```diff
@@ -30,8 +30,8 @@
   private static final Duration EXPORTER_SCHEDULE_DELAY = Duration.create(5, 0);
 
   private final SpanExporterImpl spanExporter;
-  private final RunningSpanStoreImpl runningSpanStore;
-  private final SampledSpanStoreImpl sampledSpanStore;
+  @Nullable private final RunningSpanStoreImpl runningSpanStore;
```

Comment:
Maybe I missed here when I was implementing the NoopRunningSpanStore and NoopSampledSpanStore - I should have changed them to return noop when `supportInProcessStores` is false. 

We can add `@Nullable` here now and remove them later in another PR.

Reply:
Yes, I noticed that inconsistency when running the checker framework.  I tried to fix it quickly by returning no-op classes, but some code expected the values to have type RunningSpanStoreImpl and SampledSpanStoreImpl.  I'm not sure how hard it would be to refactor.

Reply:
I opened an issue: #914